app/javascript/components/student/tracks-list/Header.jsx

```diff
@@ -1,14 +1,28 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import pluralize from 'pluralize'
+import { Loading } from '../../common/Loading'
 
-export function Header({ data, query = {} }) {
+export function Header({ latestData, query = {} }) {
+  const [text, setText] = useState()
```

Comment:
I don't know if this makes sense, but would initializing this with a default value also help reduce a re-render?

```suggestion
  const [text, setText] = useState("Exercism's language tracks")
```

Reply:
This is a good idea I think.

Reply:
Yeah. Good idea!